News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderDetailFragment.java

```diff
@@ -338,6 +352,86 @@ protected void onPostExecute(List<RssItem> rssItem) {
         }
     }
 
+    private class SearchInCurrentRssViewTask extends UpdateCurrentRssViewTask {
+
+        final private String searchString;
+
+        /**
+         * Display only items in current view, where Title OR Body contain the searchString
+         * @param context Context
+         * @param searchString String to search for
+         */
+        public SearchInCurrentRssViewTask(final Context context, final String searchString)
+        {
```

Comment:
bracket should be in the line where the method signature is defined (removing the line break here)